src/app/payments/helpers.ts

```diff
@@ -35,7 +35,7 @@ export const constructPaymentFlowBuilder = async <
   usdFromBtc,
   btcFromUsd,
 }: {
-  senderWallet: WalletDescriptor<S>
+  senderWallet: Wallet
```

Comment:
are we moving away from `WalletDescriptor`?

Reply:
We need `accountId` now as well in certain places https://github.com/GaloyMoney/galoy/pull/1682#discussion_r980472598